docs-chef-io/content/server/upgrades.md

```diff
@@ -46,6 +46,19 @@ Chef Infra Server 14.0 moved from Solr to Elasticsearch as its search index.
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+#### Upgrading to 14.13
+
+Chef Infra Server 14.13 supports external OpenSearch for indexing. To migrate from Elasticsearch to external OpenSearch, you must reindex and reconfigure your database after upgrading to Chef Infra Server 14.13. The duration of this operation will vary depending on your server hardware and the number of node objects on your Chef Infra Server.
+
+#### Steps To Enable External OpenSearch
+1. Set the `opensearch['external']` attribute to `true`.
+2. Set the `opensearch['external_url']` attribute to the external OpenSearch URL.
+For example:
+```bash
+opensearch['external'] = true
+opensearch['external_url'] = "http://127.0.0.1:9200"
+```
```

Comment:
These need blank spaces around the heading, list items, and code. Also ordered list items should all be 1, Hugo will give them the right number when they're rendered.

````suggestion

1. Set the `opensearch['external']` attribute to `true`.
1. Set the `opensearch['external_url']` attribute to the external OpenSearch URL.

For example:

```bash
opensearch['external'] = true
opensearch['external_url'] = "http://127.0.0.1:9200"
```
````